app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/model.scala

```diff
@@ -120,19 +135,15 @@ object TimeSeen {
 
 case class ServiceName(asString: String) extends AnyVal
 
-case class Environment(asString: String) extends AnyVal
-
-object Environment {
-  // todo - refactor to use the common model Environment. We do not expect prefixes/suffixes anymore
-  private def precedence(environment: Environment) = environment.asString.toLowerCase match {
-    // use `contains` so environments with prefixes/suffixes are sorted too
-    case x if x.contains("production")   => 0
-    case x if x.contains("externaltest") => 1
-    case x if x.contains("staging")      => 2
-    case x if x.contains("qa")           => 3
-    case x if x.contains("integration")  => 4
-    case x if x.contains("development")  => 5
-    case _                               => 6
+object WrWEnvironment {
+  private def precedence(environment: Environment): Int = environment match {
```

Comment:
I think there's an Ordering on Environment now...
WRW should just use the same one (and it will flip what's shown)